gcloud/bigquery/table.py

```diff
@@ -272,13 +233,19 @@ def partition_expiration(self, value):
         :type value: int
         :param value: partition experiation time in ms
         """
-        if not isinstance(value, int):
-            raise ValueError("must be an integer representing millisseconds")
-        try:
-            self._properties["timePartitioning"]["expirationMs"] = value
-        except KeyError:
-            self._properties['timePartitioning'] = {'type': "DAY"}
-            self._properties["timePartitioning"]["expirationMs"] = value
+        if not isinstance(value, (int, type(None))):
```

Comment:
Py3k doesn't have it!  See https://travis-ci.org/GoogleCloudPlatform/gcloud-python/builds/150735304#L214